site/content/docs/main/troubleshooting.md

```diff
@@ -74,6 +74,11 @@ Here are some things to verify if you receive `SignatureDoesNotMatch` errors:
 Velero cannot resume backups that were interrupted. Backups stuck in the `InProgress` phase can be deleted with `kubectl delete backup <name> -n <velero-namespace>`.
 Backups in the `InProgress` phase have not uploaded any files to object storage.
 
+## Backups stuck in `Deleting` phase
+
+You can force delete backups stuck in the `Deleting` phase with `kubectl delete backup <name> -n <velero-namespace>`.
```

Comment:
The backup stuck in the `Deleting` phase may be due to various reasons, we cannot just `kubectl delete backup` as it doesn't remove the data in the object storage.

The issue mentioned in https://github.com/vmware-tanzu/velero/issues/4245 is a dup of https://github.com/vmware-tanzu/velero/issues/4551 and is already fixed by https://github.com/vmware-tanzu/velero/pull/4817. So I don't think the doc here is needed any more